readme.md

```diff
@@ -80,6 +80,21 @@ InlineSelect::make('Status')->options($options)
     ->displayUsingLabels(),
 ```
 
+### Using closures as  `options()` argument
+
+Passing a closure to this method allows data to be formatting based on some specific logic.
```

Comment:
Change this sentence to read:
"You may pass a closure to the options method. It must return a key value pair array."